mod-invoice-storage/schemas/batch_voucher.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
If this is a collection then schema name should be `batch_voucher_collection.json` to be consistent with other schemas.

Reply:
it's more than just a simple collection